src/flick/lst/controllers/update_lst_controller.py

```diff
@@ -70,14 +74,14 @@ def process(self):
             if not (self._lst.is_saved or self._lst.is_watch_later):
                 self._lst.lst_name = lst_name
                 self._lst.lst_pic = lst_pic
-                self._add_collaborators(collaborator_ids)
+                self._add_collaborators(collaborator_ids, self._user)
```

Comment:
why are you passing `self._user` in? why can't you use it directly in `_add_collaborators`?